### PR TITLE
header added to html with style

### PR DESCRIPTION
**Pull request by ahmedadel56**
In the feature branch, I added a header to the HTML file and made its style besides this time I looked at concerns of the last submit, I hope you accept it

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```
Comment (dasileker) [inline .gitignore]: - [ ] Please add  the test.md correctly to `gitignore`. I do see it here but also is in you repo.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @ahmedadel56** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Remove the file and commit the create the file again and add it to gitignore. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (ahmedadel56) [reply .gitignore]: I don't know how to do it  correctly so please help me

**Comment by ahmedadel56** [comment]
I finally could add test.md correctly in gitignore file

**Pull request merged**